.rustfmt.toml

```diff
@@ -1,4 +1,15 @@
 version = "Two"
+edition = "2018"
```

Comment:
What does this specify?

Reply:
It refers to the Rust edition: https://doc.rust-lang.org/edition-guide/editions/index.html